src/getPackageStats.ts

```diff
@@ -91,30 +92,37 @@ export default async function getPackageStats(
       }),
     ])
 
-    const hasCSSAsset = builtDetails.assets.some(asset => asset.type === 'css')
-    const mainAsset = builtDetails.assets.find(
-      asset =>
-        asset.name === 'main' && asset.type === (hasCSSAsset ? 'css' : 'js')
-    )
+    const mainAssets = builtDetails.assets.filter(asset => asset.name === 'main')
 
-    if (!mainAsset) {
+    if (!mainAssets.length) {
       throw new UnexpectedBuildError(
         'Did not find a main asset in the built bundle'
       )
     }
 
+    const mainSizes = mainAssets.reduce((acc, asset) => {
+      acc.size += asset.size
+      acc.gzip += asset.gzip
+      if (asset.parse) {
+        acc.parse = acc.parse || {baseParseTime: 0, scriptParseTime: 0}
+        acc.parse.baseParseTime += asset.parse.baseParseTime || 0
+        acc.parse.scriptParseTime += asset.parse.scriptParseTime || 0
+      }
+      return acc
+    }, { size: 0, gzip: 0, parse: null as null | { baseParseTime: number, scriptParseTime: number } })
```

Comment:
I maintain a component library ([Vuetify](https://github.com/vuetifyjs/vuetify/)) that imports its own stylesheets. The way this was before would **only** include the size of the CSS instead of CSS + JS.